### PR TITLE
IPV6 initialization

### PR DESCRIPTION
**Pull request by tahsin352**

### CODE REVIEW

**Review by kaihoffman — COMMENTED** [review]
Sorry, I thought my comments were already sent through, looks like they were stuck on "pending"

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    civo (1.5.13)
+    civo (1.5.14)
```
[the remark this answers is not in the document]
Comment (tahsin352) [reply Gemfile.lock]: updated

File: app/models/civo/subnet.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module Civo
+  class Subnet < Base
```
[the remark this answers is not in the document]
Comment (tahsin352) [reply app/models/civo/subnet.rb:2]: No, all our class inherits base class. For example, object store credential doesn't inherit from object store.

File: lib/civo/version.rb [hunk 3]
```diff
@@ -1,3 +1,3 @@
 module Civo
-  VERSION = "1.5.13"
+  VERSION = "1.5.14"
```
[the remark this answers is not in the document]
Comment (tahsin352) [reply lib/civo/version.rb]: updated

File: app/models/civo/instance.rb [hunk 4]
```diff
@@ -6,7 +6,7 @@ class Instance < Base
     post :create, "/v#{ENV["CIVO_API_VERSION"] || "2"}/instances", requires: [:hostname, :size, :region],
       defaults: {public_ip: true, initial_user: "civo"}
     put :tags, "/v2/instances/:id/tags"
-    put :update, "/v2/instances/:id"
+    put :update, "/v2/instances/:id", request_body_type: :json
```
[the remark this answers is not in the document]
Comment (tahsin352) [reply app/models/civo/instance.rb:9]: API has removed this array of strings.

File: app/models/civo/network.rb [hunk 5]
```diff
@@ -2,8 +2,9 @@ module Civo
   class Network < Base
     if ENV["CIVO_API_VERSION"] == "2"
       get :all, "/v2/networks"
-      post :create, "/v2/networks", required: [:label]
-      put :update, "/v2/networks/:id"
+      get :find, "/v2/networks/:id"
+      post :create, "/v2/networks", required: [:label], request_body_type: :json
+      put :update, "/v2/networks/:id", request_body_type: :json
```
[the remark this answers is not in the document]
Comment (tahsin352) [reply app/models/civo/network.rb:7]: API has fields with array of strings. We have discussed about it on channel and then we checked it works without breaking any other things. So, we have implemented it.

**Pull request merged**